README.md

```diff
@@ -38,3 +50,32 @@ Use `-h` to see options for specifying different types of input and output.
 
 You can also use fontmake to run intermediate steps in the build process, via
 methods of the `fontmake.font_project.FontProject` class.
+
+### Updating 
+
+To update the program the process is similar to initial installation, but there are a few minor differences. 
+
+Enter the virtual environment created for initial installation:
+
+```bash
+source env/bin/activate  # activate the environment (run `deactivate` to exit)
+```
+
+Enter this repo, and pull this repo:
+
+```bash
+cd fontmake;
+git clone git@github.com:googlei18n/fontmake.git;
+```
+
+Reinstall fontmake's dependencies to their latest versions:
+
+```bash
+pip install -r requirements.txt
```

Comment:
Since the section is about "updating", you may wish to do `pip install --upgrade -r requirements.txt` (or `-U`).

If you don't specify `--upgrade`, pip will say `Requirements already satisfied` for those dependencies which are currently installed from their git repositories but which do not regularly modify their package version.

E.g. MutatorMath tells pip that its package version is 1.8, but that hasn't changed in years despite many changes to the code. So pip does not update because it must assume the requirement is already satisfied (i.e. you already have a MutatorMath==1.8 installed)

For fonttools, since the version string is computed from git metadata, pip is still able to detect that the package version is different, if the commit hash in the requirements.txt changes.